src/aasthaJJun20/Palindrome.java

```diff
@@ -0,0 +1,38 @@
+package aasthaJJun20;
+
+public class Palindrome {
+	// Program 1: Verify given number is palindrome or not.
+	void checkPalindromeNumber(int input) {
+		int result = 0, number = input;
+		while (number > 0) {
+			result = (result * 10) + (number % 10);
+			number /= 10;
+		}
+		if (result == input)
+			System.out.println("Given number " + input + " is palindrome.");
+		else
+			System.out.println("Given number " + input + " is not palindrome.");
+	}
+
+	// Program 2: Verify given String is palindrome or not.
+	void checkPalindromeString(String input) {
+		String result = "";
+		for (int index = input.length()-1; index >= 0; index--) {
+			result += String.valueOf(input.charAt(index));
```

Comment:
@AasthaJJun20 I am sure you can think of a solution without using extra variable result.